docs/data/sdks/java/index.md

```diff
@@ -198,17 +198,39 @@ client.logEvent(event);
 
 #### Events with groups
 
+--8<-- "includes/groups-intro-paragraph.md"
+
+!!! example
+    If Joe is in 'orgId' '10', then the `groupName` would be '10':
+
+    ```java
+    Event event = new Event("$identify");
+
+    JSONObject groupProps = new JSONObject();
+    try {
+        groupProps.put("orgId", 10);
+    } catch (JSONException e) {
+        e.printStackTrace();
+        System.err.println("Invalid JSON");
+    }
+
+    event.groupProperties = groupProps;
+    client.logEvent(event);
+    ```
+
 You can also use `logEvent` to set event-level groups. With event-level groups, the group designation applies only to the specific event being logged, and doesn't persist on the user.
 
 ```java
-Event event = New Event('Button Clicked', 'test_user_id');
+Event event = New Event('event type');
```

Comment:
In server SDKs like Java, users are required the user_id or device_id for each event. 
```suggestion
Event event = New Event('event type', 'test_user_id');
```

Reply:
fixed